cartridge.post-build

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
Either remove stub files or update them from the latest cartridge-cli package

Reply:
@akudiyar Thank you, I have removed the files, I think its a right option.